### DISCUSSION TITLE
LosslessCut and GOP question

### DISCUSSION THREAD

**Discussion by andiandi13** [mifi/lossless-cut#824] [Q&A]
I wondered if the cut was GOP accurate (on IDR keyframe only), either in normal cut mode or nearest keyframe cut mode ?

Because on specific videos, it's impossible to know, AFAIK, if it's open gop or not.

I'm using LosslessCut 2.4.0 btw.

**Comment by mifi** [comment]
I think it depends on the codec/format. I don't know the specifics of ffmpeg. Need to refer to ffmpeg source code.
